CODEOWNERS

```diff
@@ -1 +1 @@
-* @aziontech/team-product-4
+* @aziontech/team-product-4 @aziontech/team-platform-data
```

Comment:
```suggestion
* @aziontech/team-product-4 
* @aziontech/team-platform-data
```

Reply:
I've seen both working actually. 
I had a problem in the past were it would only validate after merging... don't know if that's the case though!

Reply:
It makes sense, it might be that, I commented because of the alert he gave. 😄